external/fv3fit/fv3fit/emulation/models/transformed_model.py

```diff
@@ -63,9 +63,7 @@ def build(
         factory = FieldFactory(
             self.timestep_increment_sec, self.normalize_key, self.enforce_positive, data
         )
-        model = TransformedModel(
```

Comment:
Removed `transform` from this low level object since it shouldn't need to know this.